<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SoftFileExchangerClass</class>
 <widget class="QMainWindow" name="SoftFileExchangerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>330</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SoftFileExchanger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>330</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>330</height>
    </size>
   </property>
   <widget class="QLabel" name="path_send_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Каталог для файлов на передачу</string>
    </property>
   </widget>
   <widget class="QLabel" name="IP_adress_lable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP адрес сервера получателя</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_send_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="path_send_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="path_recv_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Каталог для принимаемых файлов</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path_recv_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="path_recv_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="file_maska_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Маска передаваемых файлов</string>
    </property>
   </widget>
   <widget class="QLabel" name="block_size_label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Размер блока</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_maska_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="block_size">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>220</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="server_start_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить работу сервера для приема файлов</string>
    </property>
   </widget>
   <widget class="QPushButton" name="client_start_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Запустить процесс передачи файлов на сервер</string>
    </property>
   </widget>
   <widget class="QLabel" name="Main_lable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Конфигурация программы&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="log_lable">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>481</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Лог работы программы</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="log_data">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>481</width>
      <height>281</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="port_send_lable">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Порт приёма &lt;br/&gt;или передачи&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IP_edit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="port_edit">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>70</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignHCenter</set>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>IP_edit</zorder>
   <zorder>port_edit</zorder>
   <zorder>IP_adress_lable</zorder>
   <zorder>path_send_label</zorder>
   <zorder>path_send_edit</zorder>
   <zorder>path_send_button</zorder>
   <zorder>path_recv_label</zorder>
   <zorder>path_recv_edit</zorder>
   <zorder>path_recv_button</zorder>
   <zorder>file_maska_label</zorder>
   <zorder>block_size_label</zorder>
   <zorder>file_maska_edit</zorder>
   <zorder>block_size</zorder>
   <zorder>server_start_button</zorder>
   <zorder>client_start_button</zorder>
   <zorder>Main_lable</zorder>
   <zorder>log_lable</zorder>
   <zorder>log_data</zorder>
   <zorder>port_send_lable</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SoftFileExchanger.qrc"/>
 </resources>
 <connections/>
</ui>
